docs/docs/enclave-configuration.md

```diff
@@ -316,184 +316,72 @@ The path should be absolute or relative to the root of the enclave module.
     when building on Windows and macOS platforms. Additionally, on Windows, paths must use forwardslashes rather than
     the usual backslashes.
 
-## Assisted Configuration of Native Image Builds
+## Assisted configuration of native image builds
 
-One way of generating the reflection and serialization configuration files is by using
-[native-image-agent](https://www.graalvm.org/reference-manual/native-image/BuildConfiguration/#assisted-configuration-of-native-image-builds).
-The agent will track usage of dynamic features and generate the configuration files when run
-against a regular Java VM.
+You can generate the reflection and serialization configuration files by using the
+[native-image-agent](https://www.graalvm.org/22.0/reference-manual/native-image/Agent/#assisted-configuration-of-native-image-builds).
+This command will track the use of dynamic features and generate the configuration files when run against a regular 
```

Comment:
This bit has been copied from Graal link, we might need to make it clearer as it is now in the context of Conclave.
First of all, this is not a "command", it is more a tool. But I would keep saying "the agent".
Something like
`When using mock mode and running against a regular Java VM, the agent will track usage of dynamic features and generate the configuration files`.

Reply:
Done.